gabbi/case.py

```diff
@@ -193,14 +196,28 @@ def _clean_query_value(self, value):
         value = value.encode('UTF-8')
         return value
 
-    def _environ_replace(self, message):
+    @staticmethod
+    def _regex_replacer(replacer, escape_regex):
```

Comment:
Hmmm, yeah, I get why this is required, but the added indirection is a bummer.

Reply:
It'd be possible to rewrite all of the _replacer() functions to take an `escape_regex` argument and use `functools.partial()` instead, but I figured that wasn't actually any better.